tensorflow/core/kernels/stateful_random_ops_gpu.cu.cc

```diff
@@ -59,9 +60,13 @@ __global__ void FillKernel(
 template <typename Distribution>
 void UpdateVariableAndFill_Philox<GPUDevice, Distribution>::operator()(
     OpKernelContext* ctx, const GPUDevice& d, Distribution dist,
-    int64 output_size, int64 alg_tag_skip, ScopedUnlockUnrefVar* not_used,
-    Tensor* state_tensor,
+    UpdateVariableAndFill_Philox_Arg* arg,
     typename Distribution::ResultElementType* output_data) {
+  int64 output_size = arg->output_size;
+  int64 alg_tag_skip = arg->alg_tag_skip;
+  Tensor* state_tensor = arg->state_tensor;
+  printf("UpdateVariableAndFill_Philox out=%lld alg_tag=%lld state_tensor %p\n", output_size, alg_tag_skip, state_tensor);
+  //OP_REQUIRES(ctx, state_tensor!=0, errors::InvalidArgument("Null state tensor"));
```

Comment:
Spacing looks a bit strange here. Could you take a look at `clang-format`? It is immediately in autobuild docker images for me.

BTW, will need to remove the printf and the line follows it.